Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -1376,18 +1424,23 @@ def long_running_loop():
     certificate: str = demisto.params().get('certificate', '')
     private_key: str = demisto.params().get('key', '')
 
+    certificate_path = str()
+    private_key_path = str()
+
     try:
         port_mapping: str = PARAMS.get('longRunningPort', '')
+        port: int
         if port_mapping:
-            port: int = int(port_mapping.split(':')[0])
+            if ':' in port_mapping:
+                port = int(port_mapping.split(':')[0])
```

Comment:
I think we should be taking the second port in this case and not the first. docker port is the second argument. First one is the host port.
```suggestion
                port = int(port_mapping.split(':')[1])
```